flavors/linuxlocal/local_flavor.go

```diff
@@ -28,9 +31,12 @@ import (
 // Flavor glues together multiple plugins to mange VPP and linux interfaces configuration using local client.
 type Flavor struct {
 	injected         bool
-	Generic          generic.Flavor
-	Etcd             etcdv3.Plugin
+	Logrus           logrus.Plugin
 	LinuxLocalClient localclient.Plugin
+	HTTP             httpmux.Plugin
```

Comment:
If HTTP will be removed, LogManager won't be able to register HTTP handlers.

Linuxlocal flavor is this one.  If you meant local/local_flavor, it injects both, Base and Generic so it's the same as the vpp flavor except kafka and linuxplugin